src/WebApiToTypeScript/Endpoints/AngularEndpointsService.cs

```diff
@@ -72,6 +93,13 @@ public void WriteServiceObjectToBlock(TypeScriptBlock serviceBlock, WebApiContro
                     var interfaceWithCallFullName = $"{Config.EndpointsNamespace}.{webApiController.Name}.I{actionName}WithCall";
                     var endpointFullName = $"{Config.EndpointsNamespace}.{webApiController.Name}.{actionName}";
 
+                    var cachedBlock = Config.EndpointsSupportCaching &&
```

Comment:
That won't work because Fluent way of writing this will end the block and the function will be outside of proper scope